### ISSUE TITLE
vstack and hstack not working as intended

### ISSUE THREAD

**Issue by andrewhill157** [marimo-team/marimo#1291]
### Describe the bug

It doesn't seem like the behavior of vstack and hstack is working from version 0.4.7 to 0.4.9 as far as I can tell. In the example below only the first chart in the stack (whether it is an hstack or vstack) is visible. Sorry if I'm missing something and thanks for the help!

![image](https://github.com/marimo-team/marimo/assets/4791811/af483bb2-6697-4aad-b4d0-7664f9ec59b1)

### Environment

requirements.txt
```
altair==5.3.0
arrow==1.3.0
marimo==0.4.9
numpy==1.26.4
pandas==2.2.2
polars==0.20.23
pyarrow==16.0.0
black==24.4.2
```

### Code to reproduce

```
import marimo as mo
import altair as alt
import polars as pl

counts = pl.DataFrame(
    {
        "category": ["C1", "C2", "C3", "C4", "C5", "C6", "C7", "C8", "C9", "C10", "C11"],
        "count": [10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110],
    }
)

chart1 = (
    alt.Chart(counts.to_pandas())
    .encode(
        y="count",
        x=alt.X(
            "category",
        ),
    )
    .mark_bar(color="blue")
)

chart2 = (
    alt.Chart(counts.to_pandas())
    .encode(
        y="count",
        x=alt.X(
            "category",
        ),
    )
    .mark_bar(color="red")
)

mo.vstack(
    [
        chart1,
        chart2
    ]
)
```

**Comment by andrewhill157** [comment]
A simpler example without plots does seem to display properly in case that's helpful info:
```
import marimo as mo

mo.vstack(
    [
        mo.md("shown"),
        mo.md("also shown")
    ]
)```

![image](https://github.com/marimo-team/marimo/assets/4791811/6046764c-9e41-4cf2-84b7-a5adab301694)

**Comment by mscolnick** [comment]
agh i made a pretty sneaky mistake - pushing up a fix now and will cut a release

**Comment by andrewhill157** [comment]
awesome, thank you!

**Comment by mscolnick** [comment]
Fix is in `0.4.10` - thanks for the help reporting this
